application/ui/src/features/inspect/toolbar/sources/source-list/source-icon/source-icon.component.tsx

```diff
@@ -0,0 +1,25 @@
+import { ImagesFolder, IpCamera, VideoFile, Webcam } from '@geti-inspect/icons';
+
+interface SourceIconProps {
+    type: string;
+}
+
+export const SourceIcon = ({ type }: SourceIconProps) => {
+    if (type === 'webcam') {
```

Comment:
General: you could also use a `switch` in these type of cases.

I don't have a strong opinion on this though, so feel free to keep it as is.